### DISCUSSION TITLE
Corfunc plot functionality

### DISCUSSION THREAD

**Discussion by lucas-wilkins** [SasView/sasview#2188] [General]
My recent PR #2106 has made the Corfunc behave a bit more normally, however, they are still quite different from the other plots.

There are ongoing questions about how they should ultimately behave,
1) Discuss what should be done at this stage:
  a) how do we think the desired behaviour should be?
  b) what are the plans for other plotting functionality? There seem to be lots of deficiencies in how plots work outside of corfunc, so maybe being different isn't entirely bad.
     i) how should all the plots behave?
     ii) what's the timescale?
  c) how much should be make them consonant?
    i) also how?
    ii) also when?
2) Implement

**Comment by lucas-wilkins** [comment]
I've not really got any answer to this, will be leaving things as they are for now.

**Comment by butlerpd** [comment]
@lucas-wilkins I personally think these are great questions we **should** address but not sure about enough "community bandwidth" a the moment? Maybe this would be a good discussion topic (or subtopic ) for the code camp?
